clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -1627,7 +1627,7 @@ private DescribeAclsRequest createListAclsRequest() {
     }
 
     private DescribeAclsResponse createDescribeAclsResponse() {
-        return new DescribeAclsResponse(0, ApiError.NONE, Collections.singleton(new AclBinding(
+        return DescribeAclsResponse.prepareResponse(0, ApiError.NONE, Collections.singleton(new AclBinding(
```

Comment:
Since this is a serialization/deserialization test, I think it should not rely on `prepareResponse`.